Platform/IdavilleBoardPkg/Library/AspeedGfxLib/ast_mode.c

```diff
@@ -575,6 +575,7 @@ static void ast_crtc_dpms(struct drm_crtc *crtc, int mode)
         break;
     case DRM_MODE_DPMS_STANDBY:
         ch = 0x01;
+        break;
```

Comment:
"break" should not be added. the original logic want to program 01 in line 595. adding "break" would do nothing.